EIPS/eip-3074.md

```diff
@@ -0,0 +1,179 @@
+
+Besides better compatibility with AA, a precompile is a much less intrusive change than a new transaction type. This approach requires no changes in existing wallets, and little change in other tooling.
+
+`CALL_PRECOMPILE`'s single purpose is to set `msg.sender`. It implements the minimal functionality to enable sender abstraction for sponsored transactions. This single mindedness makes `CALL_PRECOMPILE` significantly more composable with existing Ethereum features.
+
+More logic can be implemented around the call into `CALL_PRECOMPILE`, giving more control to relayers and sponsors without sacrificing security or user experience for sponsees.
+
+### Nonces
+
+Other nonce schemes either do not provide enough security, or are too inefficient/inconvenient to be practical.
+
+ - Use sponsor nonce: every transaction from the sponsor's account invalidates every transaction-like package.
+ - Use relayer nonce: every sponsored transaction from the relayer's account invalidates every other transaction-like package. Also interacts with `SELFDESTRUCT`.
+ - Use sponsee nonce: the sponsee could attack the sponsor by submitting another transaction with a conflicting nonce at a higher gas price.
+
+Instead, by creating an independent nonce per relayer-sponsee pair, we get some attractive properties:
+ - A transaction package can only be invalidated if both the relayer and sponsee cooperate, which is nice for EOA sponsors, and necessary for AA contracts.
+ - The `SELFDESTRUCT` operation doesn't introduce replay attacks.
+
+## Backwards Compatibility
+
+No known issues.
+
+## Test Cases
+
+TODO
+
+## Implementation
+
+TODO
+
+<!--
+The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
+-->
+
+## Security Considerations
+
+ - First precompiles that require persistent storage.
+ - Potential impersonation attacks if there is a bug in the signature verification.
+ - Potential replay-attack problems if there is a bug in the replay protection, or if two chains share chain ids.
+
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+## Acknowledgments
+
+Largely inspired by [EIP ? - CallWithSigner as a potential fix for the msg.sender problem][EIP-?] and [EIP-2711].
+
+[EIP-2711]: ./eip-2711.md
+[EIP-2733]: ./eip-2733.md
+[EIP-2938]: ./eip-2938.md
+[EIP-115]: ./eip-155.md
+[EIP-?]: https://ethereum-magicians.org/t/eip-callwithsigner-as-a-potential-fix-for-the-msg-sender-problem/4340
+[gsn]: https://www.opengsn.org/
```

Comment:
Can these be inlined instead of included as references?  Most of these links are quite short, and I think the one long one will be removed with the removal of acknowledgements.  This would be for repository wide stylistic reasons more than anything, as the *vast* majority of EIPs just inline external links.